### DISCUSSION TITLE
Remove ALL teleport options when a STASH unit has not been refilled

### DISCUSSION THREAD

**Discussion by Mblair1** [runelite/runelite#17342] [Ideas]
Have a plugin that when you get an emote clue and remove the items from a STASH unit, all teleport options are removed (spellbook "cast" has been removed, jewelry rubs, cape teleports, EVERYTHING) until the items have been placed back into the STASH unit.

Kind of like how Akkha removed the "proceed" option on the exit until you deposite the pickaxe back

**Comment by Mblair1** [comment]
This is marked "should-be-external" What plugin provides this functionality?

**Reply by Nightfirecat** [reply]
None yet, the tag is an indicator that if someone makes a plugin which does this, it should go on the plugin hub rather than into the core client
